pkg/reconcilers/deploymentconfig.go

```diff
@@ -151,3 +159,53 @@ func DeploymentConfigEnvVarReconciler(desired, existing *appsv1.DeploymentConfig
 	}
 	return update
 }
+
+func findDeploymentTriggerOnImageChange(triggerPolicies []appsv1.DeploymentTriggerPolicy) (int, error) {
+	result := -1
+	for i := range triggerPolicies {
+		if triggerPolicies[i].Type == appsv1.DeploymentTriggerOnImageChange {
+			if result != -1 {
+				return -1, fmt.Errorf("found more than one imageChangeParams Deployment trigger policy")
+			}
+			result = i
```

Comment:
```suggestion
			result = i
			break
```
Minor, should there be a `break` here once the correct DeploymentTrigger is found ? :thinking:

Reply:
good. Adding the break disables detecting multiple triggers on image change, so I will remove the error above as well.

Reply:
done